tests/e2e/pageObjects/pages/Wallets.ts

```diff
@@ -179,23 +187,53 @@ const Wallets = () => {
   const importWallet = ImportWallet();
   const scanQrCodeScreen = ScanQrCodeScreen();
 
-  async function createWallet(
-    type: 'Standard HD P2SH' | 'Standard P2SH' | 'Standard HD SegWit',
-    name: string,
-    options: undefined,
-  ): Promise<void>;
-  async function createWallet(type: '2-Key Vault', name: string, options: TwoKeyWalletOptions): Promise<void>;
-  async function createWallet(type: '3-Key Vault', name: string, options: ThreeKeyWalletOptions): Promise<void>;
-  async function createWallet(type: WalletType, name: string, options: unknown): Promise<void> {}
-
-  async function importExistingWallet(
-    type: 'Standard HD P2SH' | 'Standard P2SH' | 'Standard HD SegWit',
-    name: string,
-    options: undefined,
-  ): Promise<void>;
-  async function importExistingWallet(type: '2-Key Vault', name: string, options: TwoKeyWalletOptions): Promise<void>;
-  async function importExistingWallet(type: '3-Key Vault', name: string, options: ThreeKeyWalletOptions): Promise<void>;
-  async function importExistingWallet(type: WalletType, name: string, options: unknown): Promise<void> {}
+  async function createWallet(options: CreateWalletOptions): Promise<void> {
+    const { type, name, fastPublicKey, cancelPublicKey } = options;
+
+    await dashboardScreen.tapOnAddButton();
+    await addNewWallet.createScreen.typeName(name);
+    await addNewWallet.createScreen.chooseType(type);
+    await addNewWallet.createScreen.tapOnCreateButton();
+
+    if (type === '3-Key Vault') {
```

Comment:
[?] What do you think to move all 'if' into the lower layer of tests?

Reply:
It's not a test but a method in Page Object - it allows to create or to import a wallet by invoking a single function. Primary it's for `beforeEach` purposes

Reply:
Yep, you have right. I checked in the wrong file 🤦